config/tileset/elevation.json

```diff
@@ -0,0 +1,514 @@
+{
+  "id": "ts_elevation",
+  "name": "elevation",
+  "type": "raster",
+  "description": "elevation_description",
```

Comment:
Might need a better description here.

Reply:
the aerial layer has it as "Aerial Imagery Basemap" so not exactly useful there either :P